js/src/lib/Details/Cdn.js

```diff
@@ -0,0 +1,64 @@
+import React, { Component } from 'react';
+import { Di } from './';
+
+const JsDelivr = ({ name }) => (
+  <Di
+    title="jsDelivr"
+    description={
+      <a
+        href={`https://cdn.jsdelivr.net/npm/${name}/`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        cdn.jsdelivr.net/npm/{name}/
+      </a>
+    }
+  />
+);
+
+const Unpkg = ({ name }) => (
+  <Di
+    title="unpkg"
+    description={
+      <a
+        href={`https://unpkg.com/${name}/`}
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        unpkg.com/{name}/
+      </a>
+    }
+  />
+);
+
+const BundleRun = ({ name }) => (
+  <Di
+    title="bundle.run"
+    description={
+      <a
+        href={`https://bundle.run/${name}`}
```

Comment:
Could we add the current version of the package here? The template is `https://bundle.run/${name}@${version}`.

This will avoid to unconsciously always query the latest version of the package. We can probably apply this to other CDN as well.

Reply:
Sure, we already know the latest version so I don't see why not.